### DISCUSSION TITLE
Deleted HDD while plotting

### DISCUSSION THREAD

**Discussion by Zillos** [Chia-Network/chia-blockchain#11616] [Support]
Hello Everyone

I'm having issues plotting

I was ploting on a certain HHDD via Madmax, but when the plot was created and trying to copy from temporal hdd to destined hdd, the destined HDD suddenly dissapeared. I tried copying manually the created plots to another HDD, and the same happens, the hdd suddenly dissapeared... what's wrong?

**Comment by loppefaaret** [comment]
are these destination drives externals ? it might be a bad cable, or a power issue ?
